src/main/java/io/beanmapper/config/OverrideField.java

```diff
@@ -34,7 +35,7 @@ public T get() {
             return null;
         }
         if (this.value == null) {
-            this.value = BeanMapperPerformanceLogger.runTimed("%s#%s -> %s#%s".formatted(this.getClass().getSimpleName(), "get(void)", this.getClass().getSimpleName(), "get(void)"), this.supplier);
+            this.value = BeanMapperPerformanceLogger.runTimed(LOGGING_STRING, this.supplier);
```

Comment:
Je bent nu wel wat log informatie kwijt, is dat geen probleem?

Reply:
In dit specifieke geval is het geen probleem, maar voor de CollectionHandlers is het onhandig om de informatie over welke handler het precies aangeroepen heeft te verliezen. Ik ga daar nog even aan sleutelen.